CHANGELOG.md

```diff
@@ -58,9 +61,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   `jj log`, and `jj obslog`) now accept `--types` to show only the type of file
   before and after.
 
-* `jj hide` (alias for `jj abandon`) is no longer available. Use `jj abandon`
-  instead.
-
 * `jj debug completion`, `jj debug mangen` and `jj debug config-schema` have
```

Comment:
nit: `jj debug completion` can also be moved to breaking change

Reply:
I think I didn't consider changing a debug command a breaking change, but I don't mind changing it. Done.

Reply:
`jj debug completion` was documented, so I think it's effectively a breaking change. For a debug command in general, I agree.